strongbox-user-management/src/main/java/org/carlspring/strongbox/users/security/SecurityTokenProvider.java

```diff
@@ -84,6 +84,41 @@ public String getToken(String subject,
         return jws.getCompactSerialization();
     }
 
+    public String getSubject(String token)
+    {
+
+        JwtClaims jwtClaims = getClimes(token);
```

Comment:
This method's name has a typo. It should be `getClaims`. Could you please correct this?